build/pmm-server-docker/playbook-init.yml

```diff
@@ -36,6 +36,36 @@
         regexp: '^(\s*)listen'
         replace: '\1#listen'
 
+    - name: PMM                        | Fix nginx config
```

Comment:
@adivinho we have moved all playbooks to [percona-images](https://github.com/Percona-Lab/percona-images/tree/master/ansible/roles) repo. So please move all your changes connected with playbooks to it

Reply:
https://github.com/Percona-Lab/percona-images/pull/37